test/models/policy_host_test.rb

```diff
@@ -5,4 +5,65 @@
 class PolicyHostTest < ActiveSupport::TestCase
   should belong_to(:policy)
   should validate_presence_of(:host)
+
+  setup do
+    account1 = FactoryBot.create(:user).account
+    account2 = FactoryBot.create(:user).account
+    v1 = SupportedSsg.by_os_major['7'].first
+    bm1 = FactoryBot.create(
+      :benchmark,
+      version: v1.version,
+      os_major_version: '7'
+    )
+
+    v2 = SupportedSsg.by_os_major['7'].last
```

Comment:
Same here, I'd suggest you to mock the `SupportedSsg` completely. You'll find examples at other places.